.github/workflows/publish.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  build-tests:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '3.1.100'
+      - name: Build
+        run: dotnet build --configuration Release
+      - name: Run unit tests
+        run: dotnet test --no-build --configuration Release
+
+  publish-nuget:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '3.1.x'
+      - name: Install dependencies
+        run: dotnet restore
+      - name: Build
+        run: dotnet build --configuration Release --no-restore
+      - name: Publish to NuGet
+        uses: brandedoutcast/publish-nuget@v2
+        with:
+          PROJECT_FILE_PATH: Bynder/Sdk/Bynder.Sdk.csproj
+          VERSION_REGEX: '^\s*<PackageVersion>(.*)<\/PackageVersion>\s*$'
+          TAG_FORMAT: '*'
+          NUGET_KEY: ${{secrets.NUGET_API_KEY}}
```

Comment:
Is this already working? Or sort of a "to do" thing?

Reply:
No this is working, before merging though we need to add the secrets to github account which I mentioned in the stand up today.

Reply:
And also add an API key to nuget I presume? I think only Roald can do that.

Reply:
Yes correct I reached out to him about it. Once we have the PR ready to be merged right before merging we can add these secrets to do the groundwork